wiki/rogue/performance.md

```diff
@@ -0,0 +1,4 @@
+Some performance marks about ne.rogue:
```

Comment:
You could use the printinfo.cpp file to display the results of the performance specs of rogue. And then you could break down what the table outputs. Sort of link the two together.